src/bootstrap/src/utils/cc_detect.rs

```diff
@@ -221,15 +221,19 @@ fn default_compiler(
         }
 
         t if t.contains("-wasi") => {
-            let root = build
-                .wasi_sdk_path
-                .as_ref()
-                .expect("WASI_SDK_PATH mut be configured for a -wasi target");
+            let root = build.wasi_sdk_path.as_ref();
```

Comment:
If the WASI SDK path is missing here, let's just print a warning like "WARNING: WASI_SDK_PATH is not configured, falling back to default C/C++ compiler". If we're on CI (`build.config.is_running_on_ci`), then panic instead with the original error message.

The case where both the SDK and the root are missing should be handled elsewhere in bootstrap, not here, and it is already handled when copying compiler artifacts, so I think it should be fine.